recipes/avalanche-lib/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "avalanche-lib" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/avalanche-lib-{{ version }}.tar.gz
+  sha256: 258ff56ccbe5dba4f28d0306cb87c3aeccaf4de98ff4fb3c2bc425c0594f50d9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.11
+  run:
+    - gdown
+    - gputil
+    - matplotlib-base
+    - numpy
+    - psutil
+    - python >=3.6,<3.11
+    - pytorchcv
+    - quadprog
+    - scikit-learn
+    - setuptools <=59.5.0
```

Comment:
does this really need setuptools at runtime (and with these restrictions?)

Maybe better to move that dep to `host`?